source/tutorials/integration-testing-using-virtual-machines.rst

```diff
@@ -0,0 +1,228 @@
+Integration testing using Virtual Machines
+==========================================
+
+One of the most powerful features in the Nix ecosystem is **the ability
+to provide a set of declarative NixOS configurations and use a simple
+Python interface** to interact with them using `QEMU <https://www.qemu.org/>`_
+as the backend.
+
+They are also known as **NixOS tests**.
```

Comment:
Maybe it'd be worth pointing out that those tests don't require using NixOS as the host system?

Something in terms of:

> Those tests are widely used to ensure that NixOS works as intended, so in general they are called **NixOS tests** - though they can be written and launched outside of NixOS, on any Linux machine (with MacOS support coming soon).